02-peer-protocol.md

```diff
@@ -1159,6 +1164,10 @@ whichever is greater.
 
 Note that `push_msat` has been omitted.
 
+The sending node may require the other participant to only use confirmed inputs.
```

Comment:
`require the other participant` -> if either side asks for it then it should be a requirement on both sides?